login/README.md

```diff
@@ -45,7 +48,13 @@ This Builds the app for production to the `build` folder.<br>
 It correctly bundles React in production mode and optimizes the build for the best performance.<br>
 The build is minified and the filenames include the hashes.<br>
 6. The built files may be hosted on any webserver of your choosing.
-6. To see the running Reference Facebook Chatbot Login Page, navigate to `http://localhost:9000/auth/` .
+7. To see the running Reference Facebook Chatbot Login Page, navigate to `http://localhost:9000/auth/` .
```

Comment:
```suggestion
7. To see the running Reference Chatbot Login Page, navigate to `http://localhost:9000/auth/` .
```

Reply:
Please rename `Reference Facebook Chatbot` to `Reference Chatbot` in the documentation.